specification/offazurespringboot/resource-manager/Microsoft.OffAzureSpringBoot/preview/2023-01-01-preview/springbootdiscovery.json

```diff
@@ -1354,7 +1452,7 @@
         }
       ],
       "properties": {
-        "tags": {
+        "labels": {
```

Comment:
According to your description "I had modified the tags under property to labels", the `labels` should be placed in the `properties` bag. If I understand it correctly, the `labels` definition should be in the `springbootserversProperties` instead of the same level with it. Please double check and update other places.

Reply:
Thanks, mistake for wrong place, just corrected them.